test1.js

```diff
@@ -3,3 +3,9 @@
 if(foo  === "bar") {
   console.log( 'hey' )
 }
+
+var fooBar = '' ;
+
+if(fooBar                 === "bar") {
+  console.log( 'heyyy' )
```

Comment:
Codacy found an issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=26135)
```suggestion
  console.log( "heyyy" )
```